nifi-mock/src/main/java/org/apache/nifi/util/MockFlowFile.java

```diff
@@ -290,4 +291,18 @@ public long getLineageStartIndex() {
     public long getQueueDateIndex() {
         return 0;
     }
+
+    public boolean isAttributeEqual(final String attributeName, final String expectedValue) {
+        // unknown attribute name, so cannot be equal.
+        if (attributes.containsKey(attributeName) == false)
+            return false;
+
+        String value = attributes.get(attributeName);
+        return Objects.equals(expectedValue, value);
+    }
+
+    public boolean isContentEqual(String excpected) {
+        final String value = new String(this.data, Charset.forName("UTF-8"));
+        return Objects.equals(excpected, value);
```

Comment:
Actually I'm confused why this is added when "assertContentEquals"[1] and it's various overloading methods exist.

[1] https://github.com/apache/nifi/blob/master/nifi-mock/src/main/java/org/apache/nifi/util/MockFlowFile.java#L233-L233